modules/storage/src/main/scala/zio/gcp/storage/Storage.scala

```diff
@@ -1,13 +1,307 @@
 package zio.gcp.storage
 
+import java.net.URL
+import java.util.concurrent.TimeUnit
+
+import com.google.api.gax.paging.Page
+import com.google.cloud.storage.HmacKey.HmacKeyState
+import com.google.cloud.storage.Storage._
+import com.google.cloud.storage.{ Acl, Blob, BlobId, BlobInfo, BucketInfo, CopyWriter, StorageBatch, StorageOptions }
+import com.google.cloud.{ Policy, ReadChannel, WriteChannel }
 import zio._
 
-trait Storage {
-  val storage: Storage.Service[Any]
-}
+import scala.jdk.CollectionConverters._
 
 object Storage {
-  trait Service[R] {
-    def create(bucketInfo: BucketInfo): URIO[R, BucketInfo]
+  trait Service {
```

Comment:
Are all these variants needed? If not, I would propose exposing only a couple of most-frequently used, and provide a "hook", to execute the other ones safely. I believe `zio-ftp` has a good example how to do this.

cc @regis-leray

Reply:
I looked at the underlying implementation of the Java library and have removed the ones which superfluous.